pkg/apis/extensions/v1alpha1/types.go

```diff
@@ -68,3 +69,16 @@ type Object interface {
 	// GetExtensionStatus retrieves the object's status.
 	GetExtensionStatus() Status
 }
+
+// ExtensionKinds contains all supported extension kinds.
+var ExtensionKinds = sets.NewString(
```

Comment:
Is the `DNSProvider` still working?

Reply:
Oh, indeed, that's a nice catch, thanks @vpnachev.
@timuthy we should add `DNSProvider`, `DNSEntry`, and `DNSOwner` to this list as well.

Reply:
Thanks a lot for noticing this.